### ISSUE TITLE
[Misc]: 错误输出

### ISSUE THREAD

**Issue by liuweixue001** [vllm-project/vllm-ascend#463]
### Anything you want to discuss about vllm on ascend.

按照 https://vllm-ascend.readthedocs.io/en/latest/installation.html 使用pip安装vllm
在910B上测试，
输出是错误的内容，如下：

INFO 04-03 15:14:55 llm_engine.py:436] init engine (profile, create kv cache, warmup model) took 8.58 seconds
Processed prompts: 100%|███████████████████████████| 4/4 [00:00<00:00,  4.06it/s, est. speed input: 22.31 toks/s, output: 64.91 toks/s]
Prompt: 'Hello, my name is', Generated text: 'vertisementclosed:".$ adhere sustained ה Various草肉 acheter逐渐 nextState_language recommending出品 Clause'
Prompt: 'The president of the United States is', Generated text: '挡极<=$无论是其vorandelier obligationsankerกระจายiente Eminński primitives南山 fulfil,...\n\n'
Prompt: 'The capital of France is', Generated text: '.dateFormat Kushnerintégr jane çıktıḉ.percent Sol风湿 CHAPTERSimulationunnel地产复兴isser峙'
Prompt: 'The future of AI is', Generated text: 'andalone schöne\xa0\nAuthorization-task.uint modificar artillery日至itm(prev轩辕峡谷言う importantes礞'

速度也很慢

**Comment by liuweixue001** [comment]
使用模型qwen2.5-7b 

完整日志如下：
/cog15/wxliu11/miniconda3/envs/vllm/lib/python3.10/site-packages/torch_npu/utils/collect_env.py:58: UserWarning: Warning: The /usr/local/Ascend/ascend-toolkit/latest owner does not match the current owner.
  warnings.warn(f"Warning: The {path} owner does not match the current owner.")
/cog15/wxliu11/miniconda3/envs/vllm/lib/python3.10/site-packages/torch_npu/utils/collect_env.py:58: UserWarning: Warning: The /usr/local/Ascend/ascend-toolkit/8.0.RC2.2/aarch64-linux/ascend_toolkit_install.info owner does not match the current owner.
  warnings.warn(f"Warning: The {path} owner does not match the current owner.")
INFO 04-03 15:14:25 __init__.py:30] Available plugins for group vllm.platform_plugins:
INFO 04-03 15:14:25 __init__.py:32] name=ascend, value=vllm_ascend:register
INFO 04-03 15:14:25 __init__.py:34] all available plugins for group vllm.platform_plugins will be loaded.
INFO 04-03 15:14:25 __init__.py:36] set environment variable VLLM_PLUGINS to control which plugins to load.
INFO 04-03 15:14:25 __init__.py:44] plugin ascend loaded.
INFO 04-03 15:14:25 __init__.py:198] Platform plugin ascend is activated
WARNING:root:Warning: Failed to register custom ops, all custom ops will be disabled
INFO 04-03 15:14:25 __init__.py:30] Available plugins for group vllm.general_plugins:
INFO 04-03 15:14:25 __init__.py:32] name=ascend_enhanced_model, value=vllm_ascend:register_model
INFO 04-03 15:14:25 __init__.py:34] all available plugins for group vllm.general_plugins will be loaded.
INFO 04-03 15:14:25 __init__.py:36] set environment variable VLLM_PLUGINS to control which plugins to load.
INFO 04-03 15:14:25 __init__.py:44] plugin ascend_enhanced_model loaded.
WARNING 04-03 15:14:25 registry.py:351] Model architecture Qwen2VLForConditionalGeneration is already registered, and will be overwritten by the new model class vllm_ascend.models.qwen2_vl:CustomQwen2VLForConditionalGeneration.
WARNING 04-03 15:14:25 registry.py:351] Model architecture DeepseekV2ForCausalLM is already registered, and will be overwritten by the new model class vllm_ascend.models.deepseek_v2:CustomDeepseekV2ForCausalLM.
WARNING 04-03 15:14:25 registry.py:351] Model architecture DeepseekV3ForCausalLM is already registered, and will be overwritten by the new model class vllm_ascend.models.deepseek_v2:CustomDeepseekV3ForCausalLM.
INFO 04-03 15:14:25 importing.py:16] Triton not installed or not compatible; certain GPU-related functions will not be available.
INFO 04-03 15:14:35 config.py:549] This model supports multiple tasks: {'reward', 'score', 'embed', 'generate', 'classify'}. Defaulting to 'generate'.
INFO 04-03 15:14:35 llm_engine.py:234] Initializing a V0 LLM engine (v0.7.3) with config: model='/cog15/wxliu11/liu/code/models/Qwen/Qwen2___5-7B-Instruct', speculative_config=None, tokenizer='/cog15/wxliu11/liu/code/models/Qwen/Qwen2___5-7B-Instruct', skip_tokenizer_init=False, tokenizer_mode=auto, revision=None, override_neuron_config=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.bfloat16, max_seq_len=32768, download_dir=None, load_format=auto, tensor_parallel_size=1, pipeline_parallel_size=1, disable_custom_all_reduce=False, quantization=None, enforce_eager=False, kv_cache_dtype=auto,  device_config=npu, decoding_config=DecodingConfig(guided_decoding_backend='xgrammar'), observability_config=ObservabilityConfig(otlp_traces_endpoint=None, collect_model_forward_time=False, collect_model_execute_time=False), seed=0, served_model_name=/cog15/wxliu11/liu/code/models/Qwen/Qwen2___5-7B-Instruct, num_scheduler_steps=1, multi_step_stream_outputs=True, enable_prefix_caching=False, chunked_prefill_enabled=False, use_async_output_proc=True, disable_mm_preprocessor_cache=False, mm_processor_kwargs=None, pooler_config=None, compilation_config={"splitting_ops":[],"compile_sizes":[],"cudagraph_capture_sizes":[256,248,240,232,224,216,208,200,192,184,176,168,160,152,144,136,128,120,112,104,96,88,80,72,64,56,48,40,32,24,16,8,4,2,1],"max_capture_size":256}, use_cached_outputs=False, 
/cog15/wxliu11/miniconda3/envs/vllm/lib/python3.10/site-packages/torch_npu/contrib/transfer_to_npu.py:292: ImportWarning: 
    *************************************************************************************************************
    The torch.Tensor.cuda and torch.nn.Module.cuda are replaced with torch.Tensor.npu and torch.nn.Module.npu now..
    The torch.cuda.DoubleTensor is replaced with torch.npu.FloatTensor cause the double type is not supported now..
    The backend in torch.distributed.init_process_group set to hccl now..
    The torch.cuda.* and torch.cuda.amp.* are replaced with torch.npu.* and torch.npu.amp.* now..
    The device parameters have been replaced with npu in the function below:
    torch.logspace, torch.randint, torch.hann_window, torch.rand, torch.full_like, torch.ones_like, torch.rand_like, torch.randperm, torch.arange, torch.frombuffer, torch.normal, torch._empty_per_channel_affine_quantized, torch.empty_strided, torch.empty_like, torch.scalar_tensor, torch.tril_indices, torch.bartlett_window, torch.ones, torch.sparse_coo_tensor, torch.randn, torch.kaiser_window, torch.tensor, torch.triu_indices, torch.as_tensor, torch.zeros, torch.randint_like, torch.full, torch.eye, torch._sparse_csr_tensor_unsafe, torch.empty, torch._sparse_coo_tensor_unsafe, torch.blackman_window, torch.zeros_like, torch.range, torch.sparse_csr_tensor, torch.randn_like, torch.from_file, torch._cudnn_init_dropout_state, torch._empty_affine_quantized, torch.linspace, torch.hamming_window, torch.empty_quantized, torch._pin_memory, torch.autocast, torch.load, torch.Generator, torch.set_default_device, torch.Tensor.new_empty, torch.Tensor.new_empty_strided, torch.Tensor.new_full, torch.Tensor.new_ones, torch.Tensor.new_tensor, torch.Tensor.new_zeros, torch.Tensor.to, torch.Tensor.pin_memory, torch.nn.Module.to, torch.nn.Module.to_empty
    *************************************************************************************************************
    
  warnings.warn(msg, ImportWarning)
/cog15/wxliu11/miniconda3/envs/vllm/lib/python3.10/site-packages/torch_npu/contrib/transfer_to_npu.py:247: RuntimeWarning: torch.jit.script and torch.jit.script_method will be disabled by transfer_to_npu, which currently does not support them, if you need to enable them, please do not use transfer_to_npu.
  warnings.warn(msg, RuntimeWarning)
WARNING 04-03 15:14:37 utils.py:2262] Methods add_lora,add_prompt_adapter,cache_config,compilation_config,current_platform,list_loras,list_prompt_adapters,load_config,pin_lora,pin_prompt_adapter,remove_lora,remove_prompt_adapter not implemented in <vllm_ascend.worker.worker.NPUWorker object at 0xfffd20228cd0>
<frozen importlib._bootstrap>:671: ImportWarning: TBEMetaPathLoader.exec_module() not found; falling back to load_module()
<frozen importlib._bootstrap>:914: ImportWarning: TEMetaPathFinder.find_spec() not found; falling back to find_module()
<frozen importlib._bootstrap>:671: ImportWarning: TBEMetaPathLoader.exec_module() not found; falling back to load_module()
Loading safetensors checkpoint shards:   0% Completed | 0/4 [00:00<?, ?it/s]
Loading safetensors checkpoint shards:  25% Completed | 1/4 [00:00<00:01,  2.99it/s]
Loading safetensors checkpoint shards:  50% Completed | 2/4 [00:00<00:00,  2.31it/s]
Loading safetensors checkpoint shards:  75% Completed | 3/4 [00:01<00:00,  2.22it/s]
Loading safetensors checkpoint shards: 100% Completed | 4/4 [00:01<00:00,  2.15it/s]
Loading safetensors checkpoint shards: 100% Completed | 4/4 [00:01<00:00,  2.23it/s]

mki_log log dir:/cog15/wxliu11/atb/log exist
INFO 04-03 15:14:55 executor_base.py:111] # npu blocks: 4986, # CPU blocks: 585
INFO 04-03 15:14:55 executor_base.py:116] Maximum concurrency for 32768 tokens per request: 19.48x
INFO 04-03 15:14:55 llm_engine.py:436] init engine (profile, create kv cache, warmup model) took 8.58 seconds
Processed prompts: 100%|███████████████████████████| 4/4 [00:00<00:00,  4.06it/s, est. speed input: 22.31 toks/s, output: 64.91 toks/s]
Prompt: 'Hello, my name is', Generated text: 'vertisementclosed:".$ adhere sustained ה Various草肉 acheter逐渐 nextState_language recommending出品 Clause'
Prompt: 'The president of the United States is', Generated text: '挡极<=$无论是其vorandelier obligationsankerกระจายiente Eminński primitives南山 fulfil,...\n\n'
Prompt: 'The capital of France is', Generated text: '.dateFormat Kushnerintégr jane çıktıḉ.percent Sol风湿 CHAPTERSimulationunnel地产复兴isser峙'
Prompt: 'The future of AI is', Generated text: 'andalone schöne\xa0\nAuthorization-task.uint modificar artillery日至itm(prev轩辕峡谷言う importantes礞'

**Comment by Potabk** [comment]
@liuweixue001 Can you show the Sampling parameters and torch_npu version

**Comment by Yikun** [comment]
Please install the specific pytorch: https://vllm-ascend.readthedocs.io/en/latest/installation.html#setup-vllm-and-vllm-ascend

The accuracy problem might related to wrong torch_npu or CANN version.

I will close this because no reply. Feel free to reopen if you have more question here
